config/create_genesis.ts

```diff
@@ -0,0 +1,180 @@
+import { BigNumber, ethers } from "ethers";
```

Comment:
hmm does it make sense to put this in this dir? (1) having all the scripts spread out across several directories makes it harder to figure out how to use the proj; and (2) i don't like mixing config files and scripts